task/upload.go

```diff
@@ -62,20 +62,20 @@ func handleUploadVOD(p handleUploadVODParams) (*TaskHandlerOutput, error) {
 	)
 	switch step {
 	case "":
-		// TODO: Move this input decryption logic to catalyst
-		if uploadParams := tctx.Params.Upload; uploadParams != nil {
-			var err error
-			inUrl, err = decryptInputFile(tctx, inUrl, *uploadParams)
-			if err != nil {
-				return nil, fmt.Errorf("error decrypting input file: %w", err)
-			}
-		}
 		fallthrough
 	case "rateLimitBackoff":
```

Comment:
Delete this separate `""` case since we don't need any custom logic anymore. This should be
```go
case "", "rateLimitBackoff":
```